_docs/reference/release-roadmap.md

```diff
@@ -27,6 +27,7 @@ Istio 0.2 will be our second release, and will work across multiple Kubernetes c
 ## Istio 0.3
 
 Istio 0.3 will be our third release, focused on performance, scale, and stability. The full feature set for 0.3 is still TBD, based on feedback from the community. Some of the additional features we are considering include:
+- Support for multiple clusters in a single mesh and Istio meshes without Kubernetes.
```

Comment:
Break this into two parts, they're separate.

- Support for multiple clusters in a single mesh.
- Support for Istio meshes without Kubernetes.